x/crosschain/keeper/msg_tss_voter.go

```diff
@@ -129,6 +129,32 @@ func (k msgServer) UpdateTssAddress(goCtx context.Context, msg *types.MsgUpdateT
 	return &types.MsgUpdateTssAddressResponse{}, nil
 }
 
+func (k msgServer) MigrateTssFunds(goCtx context.Context, msg *types.MsgMigrateTssFunds) (*types.MsgMigrateTssFundsResponse, error) {
```

Comment:
I think it would give more clarity to have one file per message.
Do we use `MigrateTSSFundsForChain` for another case than in this message? I think both functions should be in the same file